.github/workflows/ci.yaml

```diff
@@ -6,74 +6,98 @@ on:
 jobs:
   ci:
     runs-on: ubuntu-20.04
-    timeout-minutes: 15
+    timeout-minutes: 20
 
     strategy:
       matrix:
-        mode: [debug, release]
-        module: [builtin, loadable]
-        outputdir: [src, build]
+        arch: [x86_64, arm64]
+        toolchain: [gcc, clang, llvm]
+        config: [debug, release]
+        rustc: [nightly-2021-01-02]
+        output: [src]
+        include:
```

Comment:
`include` is added to the matrix generated by cross-product of the other options, is that right?

Reply:
Yeah, this only adds 1 more (which should be enough to keep `O=` working, no need to add another dimension to the matrix).